src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_BORDER = 5;
+    private static final int BORDER = 101;
+    private int randomFigureIndex;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
```

Comment:
Avoid beginning method implementations with an empty line. It's good practice to start the method body immediately after the method signature.